examples/game-of-life-challenge/src/lib.rs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+/*! Main library for the Game-of-Life challenge. Should compile for Wasm and Rust. */
```

Comment:
(I'd write this as "Game of Life challenge", since it's a name, not an adjective?)

Reply:
I believe that's correct